packages/plugin-core/src/commands/LaunchTutorialCommand.ts

```diff
@@ -10,6 +10,10 @@ type CommandInput = {};
 
 type CommandOutput = void;
 
+/**
+ * This command is a bit of a misnomer - it actually launches the welcome
+ * webview page
+ */
 export class LaunchTutorialCommand extends BasicCommand<
```

Comment:
can we rename it in that case?

Reply:
sure - at first I left it b/c I thought it would interfere with our existing charts, but it looks like we don't look at this.